brands/shop/confectionery.json

```diff
@@ -39,6 +39,19 @@
       "shop": "confectionery"
     }
   },
+  "shop/confectionery|Kilwins": {
+    "countryCodes": ["us"],
+    "matchTags": ["amenity/ice_cream"],
+    "tags": {
+      "alt_name": "Kilwin's",
+      "brand": "Kilwins",
+      "cuisine": "chocolate;ice_cream;popcorn",
+      "name": "Kilwins",
+      "shop": "confectionery",
+      "wikidata": "Q38250832",
```

Comment:
oh!  also these need to be `brand:wikidata` and `brand:wikipedia`.. I'll fix that too.

Reply:
Sorry about that. Thanks.